app.py

```diff
@@ -244,15 +244,24 @@ def map_state(objState):
     state = objState['state']
     socketId = request.sid
     sendData = ''
-    if state == 'NY':
-        sendData = 'User hits NY'
-    elif state == 'NJ':
-        sendData = 'User hits NJ'
-    else:
-        sendData = 'Please choose New Jersey or New York'
-
+    news_file = open('states_info.json', 'r')
+    news_json = json.load(news_file)
+    news_file.close()
+    for stateData in news_json['states']:
+        if state == stateData['stateCode']:
```

Comment:
A little confused by this. Why not have the socket emit inside the if statement? Also, why not send stateData directly instead of extracting out all these fields?